Documentation/FAQs.md

```diff
@@ -43,3 +44,39 @@ When in doubt, please check your compiler's documentation for the options necess
 
 * #### cl.exe
   Symbols are located in the `*.pdb` file.
+
+## How to enable logging
+
+Enabling logging will show communication information between VS Code and our extension and between our extension and the debugger.
+
+### Logging for `MI` debuggers
+
+The logging block with its defaults is as follows:
```

Comment:
I think it might be better to show code snippets with the values you want people to copy.  If you redirect people here, they are likely to want to do that rather than see what the defaults are.

Reply:
There are different combinations depending on what the user wants to see.